core-java-modules/core-java-lang-operators-2/src/test/java/com/baeldung/operators/AssignmentUnitTest.java

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.operators;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class AssignmentUnitTest {
+
+    @Test
+    void whenCompareTwoValuesUsingTheOperatorEqualTo_thenSuccess() {
```

Comment:
The test should talk about the assignment operator but is talking about using the equal operator.